src/client/src/rt-theme/colors.js

```diff
@@ -0,0 +1,203 @@
+import _ from 'lodash';
+import { mix, rgb, rgba } from 'polished';
```

Comment:
Can you please provide some comments on this file to describe your approach. This is a very important part of the application and developers will need to understand the core concepts

Reply:
Yes I will document the what and why with regards to colors and palettes.

Are you also suggesting I document some color theory or how the math from these libs is incorrect?